File: pt-BR/C# para criancas/ControlStructures.cs
using System;

namespace Book.For.Kids.KeyConcepts
{
    public class Program
    {
        public static void Main()
        {
            ControlStructures.Sample();
        }
    }

    /// <summary>
    /// Text in the book ;D
    /// </summary>
    public static class ControlStructures
    {
        public static void Sample()
        {
            bool planetIsLikeEarth = false;
            bool findNewPlanet = false;
            while (!findNewPlanet)
            {
				Console.Write("O planeta encontrado é parecido com a Terra? ");
                planetIsLikeEarth = Console.ReadLine() == "SIM";
                if (planetIsLikeEarth)
                {
                    findNewPlanet = true;
                    Console.WriteLine("Huston, já não temos mais um problema!");
                }
            }
        }
    }
}

File: pt-BR/JavaScript for kids/controlStructures.js
/**
  * Text in the book ;D
  */
class ControlStructures {
    static sample() {
        let planetIsLikeEarth = false;
        let findNewPlanet = false;

        while (!findNewPlanet) {
            const userInput = prompt("O planeta encontrado é parecido com a Terra? (Digite SIM ou NÃO)").toUpperCase();
            planetIsLikeEarth = userInput === "SIM";

            if (planetIsLikeEarth) {
                findNewPlanet = true;
                console.log("Huston, já não temos mais um problema!");
            }
        }
    }
}

ControlStructures.sample();
